### PR TITLE
avocado-plugins-vt.spec: add python2-simplejson for rhel7

### PR DESCRIPTION
**Pull request by xutian**
Fix issue#888 - not module named simplejson in rhel7 host

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: avocado-plugins-vt.spec [hunk 1]
```diff
@@ -21,7 +21,7 @@ Requires: python-imaging
 %if 0%{?el6}
 Requires: gstreamer-python, gstreamer-plugins-good, python-simplejson
 %else
-Requires: pygobject2, gstreamer1-plugins-good
+Requires: pygobject2, gstreamer1-plugins-good, python2-simplejson
```
Comment (clebergnu) [inline avocado-plugins-vt.spec]: @xutian according to these packaging guidelines (under section Provides):

http://fedoraproject.org/wiki/Packaging:Python

It's better to stick to `python-simplejson`, which will be provided by `python2-simplejson`:

```
rpm -q --provides python2-simplejson 
python-simplejson = 3.10.0-1.el7
python-simplejson(x86-64) = 3.10.0-1.el7
python2-simplejson = 3.10.0-1.el7
python2-simplejson(x86-64) = 3.10.0-1.el7
```

With that, it's possible to se the same python-simplejson dep for all distros:

```diff
diff --git a/avocado-plugins-vt.spec b/avocado-plugins-vt.spec
index 4dd9eda..fddbb67 100644
--- a/avocado-plugins-vt.spec
+++ b/avocado-plugins-vt.spec
@@ -15,11 +15,11 @@ Source0: https://github.com/avocado-framework/%{modulename}/archive/%{commit}/%{
 BuildRequires: python2-devel, python-setuptools
 BuildArch: noarch
 Requires: avocado >= 36.3
-Requires: python, autotest-framework, p7zip, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces
+Requires: python, autotest-framework, p7zip, tcpdump, iproute, iputils, gcc, glibc-headers, python-devel, nc, aexpect, git, python-netaddr, python-netifaces, python-simplejson
 
 Requires: python-imaging
 %if 0%{?el6}
-Requires: gstreamer-python, gstreamer-plugins-good, python-simplejson
+Requires: gstreamer-python, gstreamer-plugins-good
 %else
 Requires: pygobject2, gstreamer1-plugins-good
 %endif

```

Comment (clebergnu) [inline avocado-plugins-vt.spec]: And I've tested that such a test produces a package that brings in the right dep:

```
Dependencies Resolved

==================================================================================================================================================
 Package                        Arch               Version                         Repository                                                Size
==================================================================================================================================================
Installing:
 avocado-plugins-vt             noarch             47.0-0.el7.centos               /avocado-plugins-vt-47.0-0.el7.centos.noarch              15 M
Installing for dependencies:
 python-netaddr                 noarch             0.7.5-7.el7                     base                                                     983 k
 python-netifaces               x86_64             0.10.4-3.el7                    base                                                      17 k
 python2-simplejson             x86_64             3.10.0-1.el7                    epel                                                     188 k

Transaction Summary
==================================================================================================================================================
Install  1 Package (+3 Dependent packages)
```

**Comment by xutian** [comment]
@clebergnu,   thanks your comments, commit updated, would you like to help review it. thanks

**Comment by clebergnu** [comment]
LGTM!

**Pull request merged**